template.tpl

```diff
@@ -451,11 +473,19 @@ ___WEB_PERMISSIONS___
             "listItem": [
               {
                 "type": 1,
-                "string": "https://cdn.matomo.cloud/*/piwik.js https://cdn.matomo.cloud/*/matomo.js https://cdn.innocraft.cloud/*/piwik.js https://cdn.innocraft.cloud/*/matomo.js"
+                "string": "https://*.matomo.cloud/piwik.js"
```

Comment:
@AltamashShaikh can you confirm we will load these files through the CDN? eg https://cdn.matomo.cloud/web.innocraft.cloud/matomo.js ?

We could always use `cdn.matomo.cloud` actually and always `matomo.js` I suppose.

Reply:
it's less secure to load files directly that way, it's slower, and costs us more

Reply:
@tsteur Updated to use `https://cdn.matomo.cloud/` always.